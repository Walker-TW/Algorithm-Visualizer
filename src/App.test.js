// import React from 'react';

// import App from './App';
// import matchMediaPolyfill from 'mq-polyfill';

// import { shallow, mount, render } from 'enzyme';

// describe('<App />', () => {
//   beforeAll(() => {
//     matchMediaPolyfill(window);
//     window.resizeTo = function resizeTo(width, height) {
//       Object.assign(this, {
//         innerWidth: width,
//         innerHeight: height,
//         outerWidth: width,
//         outerHeight: height,
//       }).dispatchEvent(new this.Event('resize'));
//     };
//   });

//   it('renders', () => {
//     const component = shallow(<App />);
//     expect(component).toMatchSnapshot();
//   });

//   it('Changes thr amount of nodes based on window size', () => {
//     window.resizeTo = jest.fn();
//     window.resizeTo(375, 667);
//     const wrapper = shallow(<App />);
//     console.log(wrapper);
//     expect(wrapper.find(Node).length).toEqual(504);
//   });
// });
