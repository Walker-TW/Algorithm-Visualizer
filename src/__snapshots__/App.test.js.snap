// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders 1`] = `
<div
  className="App"
>
  <div
    className="Grid"
  >
    <div
      className="Button"
    >
      <div
        className="fade alert alert-primary show"
        role="alert"
      >
        Please Choose A Start & End Node
      </div>
    </div>
    <div
      className="Column"
    >
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
    </div>
    <div
      className="Column"
    >
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
    </div>
    <div
      className="Column"
    >
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
    </div>
    <div
      className="Column"
    >
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
    </div>
    <div
      className="Column"
    >
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
    </div>
    <div
      className="Column"
    >
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
    </div>
    <div
      className="Column"
    >
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
    </div>
    <div
      className="Column"
    >
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
    </div>
    <div
      className="Column"
    >
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
    </div>
    <div
      className="Column"
    >
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
      <div
        className="Node   "
        onClick={[Function]}
      />
    </div>
  </div>
</div>
`;
